src/views/activity/components/transaction-card/transaction-card.view.tsx

```diff
@@ -2,8 +2,8 @@ import { FC } from "react";
 import { useNavigate } from "react-router-dom";
 
 import useTransactionCardStyles from "src/views/activity/components/transaction-card/transaction-card.styles";
```

Comment:
This component should change to BridgeCard

Reply:
Right, done! I've also renamed the `TransactionForm` to `BridgeForm`.